### PR TITLE
[stable/cluster-autoscaler] allow AWS_REGION to be optional

### PR DESCRIPTION
**Pull request by rifelpet**
#### What this PR does / why we need it:

Cluster Autoscaler recently added support for detecting AWS region from metadata:

https://github.com/kubernetes/autoscaler/releases/tag/cluster-autoscaler-1.13.0

https://github.com/kubernetes/autoscaler/pull/1288

This change allows the chart user to specify a cloud provider of `aws` but not specify an `AWS_REGION` by setting `awsRegion` to an empty string.

#### Special notes for your reviewer:
N/A

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

@mgoodness

### CODE REVIEW

**Review by AlexShemeshWix — APPROVED** [review]

**Comment by davidkarlsen** [comment]
@rifelpet Sorry for the delay!

File: stable/cluster-autoscaler/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 description: Scales worker nodes within autoscaling groups.
 icon: https://github.com/kubernetes/kubernetes/blob/master/logo/logo.png
 name: cluster-autoscaler
-version: 0.12.1
+version: 0.12.2
```
Comment (davidkarlsen) [inline stable/cluster-autoscaler/Chart.yaml]:
```suggestion
version: 0.12.3
```

**Pull request merged**